src/main/kotlin/com/dp/advancedgunnerycontrol/weaponais/tags/PrioritisePDTag.kt

```diff
@@ -0,0 +1,31 @@
+package com.dp.advancedgunnerycontrol.weaponais.tags
+
+import com.dp.advancedgunnerycontrol.weaponais.bigness
+import com.dp.advancedgunnerycontrol.weaponais.isBig
+import com.dp.advancedgunnerycontrol.weaponais.isPD
+import com.dp.advancedgunnerycontrol.weaponais.isSmall
+import com.fs.starfarer.api.combat.CombatEntityAPI
+import com.fs.starfarer.api.combat.MissileAPI
+import com.fs.starfarer.api.combat.ShipAPI
+import com.fs.starfarer.api.combat.WeaponAPI
+import org.lwjgl.util.vector.Vector2f
+
+class PrioritisePDTag(weapon: WeaponAPI) : WeaponAITagBase(weapon) {
+
+    override fun isValidTarget(entity: CombatEntityAPI): Boolean {
+        return  (entity as? ShipAPI)?.let { isSmall(it) } ?: false || (isPD(weapon) && (entity as? MissileAPI) != null)
+    }
+    override fun isBaseAiValid(entity: CombatEntityAPI): Boolean = false
```

Comment:
I feel like 
```
= entity is MissileAPI
```
wouldn't hurt here. That way, if the base AI is already targeting a missile, we don't need to use the custom AI

Reply:
good pick up, yeah that would help efficiency :)